ocs_ci/templates/ocs-deployment/local-storage-optional-operators.yaml

```diff
@@ -6,11 +6,14 @@ metadata:
 spec:
   repositoryDigestMirrors:
   - mirrors:
-    - quay.io/openshift-qe-optional-operators
-    source: registry.redhat.io/openshift4
+    - brew.registry.redhat.io
```

Comment:
can you add a comment for this file to be used internally?

Reply:
@vasukulkarni do you have a comment in mind? I'm not sure what you mean

Reply:
@clacroix12 the one I have is `# This configuration should be used internally within redhat vpn ` on the top of the file

Reply:
My understanding is that this is reachable outside of the redhat network so I don't think we need that

Reply:
bit confusing atleast the last one resolves to psi address but if it works externally then we dont need comment for sure.